cg/constants/priority.py

```diff
@@ -7,6 +7,11 @@
 
 PRIORITY_OPTIONS = list(PRIORITY_MAP.keys())
 
+RSYNC_ACCOUNT_TO_QOS = {
+    "production": "normal",
+    "development": "low",
```

Comment:
And also attach it to the SlurmQos, something like:
```suggestion
    "development": SlurmQos.LOW,
```